src/components/RegForm/RegForm.js

```diff
@@ -4,10 +4,10 @@ import { registerUser } from '../../services/reg.js';
 
 function RegForm(){
     const [formData, setFormData] = useState({
-        email: '',
         username: '',
+        email: '',
         password: '',
-        password_confirm: '',
```

Comment:
why are u redoing work code? variable in the back serializer called password_confirm
explain yourself

Reply:
my bad master(((((